### PR TITLE
creation de la page discipline

### PR DESCRIPTION
**Pull request by yazidsef**
creation coté front-endpage discipline

### CODE REVIEW

File: src/Controller/DisciplineController.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Controller;
+
+class HomeController extends AbstractController
```
Comment (LaserHuy) [inline src/Controller/DisciplineController.php:5]: Change HomeController to DisciplineController

File: src/View/discipline/index.html.twig [hunk 2]
```diff
@@ -0,0 +1,139 @@
+ {% extends "../layout.html.twig" %}
```
Comment (LaserHuy) [inline src/View/discipline/index.html.twig:1]: delete "../" make to this  {% extends "layout.html.twig" %}

**Review by LaserHuy — CHANGES_REQUESTED** [review]

File: src/View/discipline/index.html.twig [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+    {% block stylesheet %}<link rel="stylesheet" href="/assets/css/discipline.css">{% endblock %}
```
Comment (LaserHuy) [inline src/View/discipline/index.html.twig:2]: {% block stylesheet %}
        {{ parent() }}
        <link rel="stylesheet" href="/assets/css/discipline.css">
{% endblock %}
Using parent() to recieve also the style sheets from layout.html.twig, replace with the code above bro. Thanks

Comment (yazidsef) [reply src/View/discipline/index.html.twig:2]: {% block stylesheet %}{{ parent() }}<link rel="stylesheet" href="/assets/css/discipline.css">{% endblock %}
like that !

**Review by LaserHuy — APPROVED** [review]
Good jobs bro

**Review by Benja-san — APPROVED** [review]

**Pull request merged**